astropy/io/ascii/latex.py

```diff
@@ -229,6 +229,53 @@ class Latex(core.BaseReader):
     row, no ``\multicolumn`` or similar constructs), specifically, it
     can read the tables that it writes.
 
+    .. note:: **Units in LaTeX tables**
+
+        The LaTeX writer will output units in the table if they are present in the
+        column info::
+
+            >>> import sys
+            >>> import astropy.units as u
+            >>> from astropy.table import Table
+            >>> t = Table({'v': [1, 2] * u.km/u.s, 'class': ['star', 'jet']})
+            >>> t.write(sys.stdout, format='ascii.latex')
```

Comment:
Small suggestion: let's use `StringIO()` instead of `sys.stdout` so that the result can be used below. I.e.,
```
import io

out = io.StringIO()
t.write(out, format='ascii.latex')
print(out.getvalue())
```
Two more lines here, but removes the need to type in the result below (and maybe leaves readers with  something that is more useful?)

Reply:
I thought about using `StingIO` and decided to go with the simpler, more beginner friendly "let's jut print it to screen". On the other hand, I appreciate the point that it teaches something else that's useful along the way, so I will change that.